pkg/ivd/ivd_protected_entity_type_manager_test.go

```diff
@@ -86,10 +92,61 @@ func getVcConfigFromParams(params map[string]interface{}) (*url.URL, bool, error
 	return &vcUrl, insecure, nil
 }
 
+// TODO: remove. Used only in test.
```

Comment:
You can remove this method as all the VC credential is available in the IVD PETM passed from the caller, e.g. velero-plugin-for-vsphere.